pkg/api/v1alpha1/kepler_types.go

```diff
@@ -98,6 +99,14 @@ type ExporterSpec struct {
 	// +kubebuilder:validation:Maximum=65535
 	// +kubebuilder:validation:Minimum=1
 	Port int32 `json:"port,omitempty"`
+
+	// Defines which Nodes the Pod is scheduled on
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// If specified, define Pod's tolerations
+	// +optional
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
```

Comment:
Just wondering if we can set the defaults here?

Reply:
in code it is fine too, not too much difference.